docs/components/connectors/react-components/_connectors-table.js

```diff
@@ -360,6 +362,20 @@ const SearchableTable = () => {
       link: "../salesforce/",
       image: SalesforceImg,
     },
+    {
+      name: "SAP OData",
+      description: "Orchestrate SAP tasks and data.",
+      type: "Outbound",
+      link: "../salesforce/",
```

Comment:
@mesellings I am trying to link to https://docs.camunda.io/docs/next/components/camunda-integrations/sap/odata-connector/ -- what would this look like for the link above?